Chummer/Utilities/frmDiceRoller.cs

```diff
@@ -101,22 +103,40 @@ private void cmdRollDice_Click(object sender, EventArgs e)
                     if (chkCinematicGameplay.Checked)
                         intTarget = 4;
 
-                    if (intResult >= intTarget)
+                    if (intResult >= intTarget) {
                         intHitCount++;
-                    if (intResult <= intGlitchMin)
+                        lviCur.ForeColor = ColorManager.DieHitFore;
+                        lviCur.BackColor = ColorManager.DieHitBackground;
```

Comment:
Make sure you incorporate hooks into the `ColorManager`'s recoloring method so that `ListViewItem` colors are properly updated if/when the user switches between Light and Dark modes while the program is open (e.g. they have it set to Automatic in Chummer and then change their preferred mode in Windows' Settings).

Reply:
The frmDiceRoller.cs is currently not updated when changing the light/dark mode. Even not when doing it via Windows Settings. This makes it hard to test and I would tend to exclude this into a separate feature request.

Current workaround is to re-open the DiceTool, which will anyway remove the previous results.